Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -137,7 +143,8 @@ public class DynamoDbDefaults
                 DimensionNames = new[] { "TableName" },
                 ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
                 Statistic = Statistic.Sum,
-                Namespace = AwsNamespace.DynamoDb
+                Namespace = AwsNamespace.DynamoDb,
+                TreatMissingData = "notBreaching"
```

Comment:
Can we create some constants rather than having these magic strings everywhere?

Reply:
Sure thing - would you think it's worth adding to the existing AwsConstants class or create a dedicated one just for the TreatMissingData values?